bot/exts/moderation/voice_gate.py

```diff
@@ -53,6 +69,13 @@ async def voice_verify(self, ctx: Context, *_) -> None:
         - You must not be actively banned from using our voice channels
         - You must have been active for over a certain number of 10-minute blocks
         """
+        # If user has received a ping in voice_verification, delete the message
+        if message_id := await self.redis_cache.get(ctx.author.id, NO_MSG):
+            with suppress(discord.NotFound):
+                self.voice_verification_channel = self.bot.get_channel(Channels.voice_gate)
+                await self.bot.http.delete_message(self.voice_verification_channel, message_id)
```

Comment:
Sorry, I may have confused you. The http method does *not* take the channel object, it takes the ID, which is why it makes this situation easier. You can do:
```py
delete_message(Channels.voice_gate, message_id)
```
Without fetching or getting the channel. The channel never needs to be retrieved in any way.

Reply:
See dbff099.